symfony/client/src/orangehrmMaintenancePlugin/components/SelectedEmployee.vue

```diff
@@ -101,7 +101,6 @@ export default {
       default: false,
     },
   },
-
   emits: ['submit'],
```

Comment:
I think better to keep blank lines here (in line 104 and line 127) just to have some separation. Looks better to me. Up to you if you keep or not